src/org/labkey/test/components/glassLibrary/grids/ResponsiveGrid.java

```diff
@@ -448,10 +448,10 @@ public void clickLink(String text)
 
     /**
      * locates the first link in the specified column, clicks it, and waits for the URL to update
-     * @param text  text for link to match
      * @param column    column in which to search
+     * @param text  text for link to match
      */
-    public void clickLink(String text, String column)
```

Comment:
for consistency.  Elsewhere we follow a key/value ordering

Reply:
Please do the same for `selectRows` and `isRowSelected`.
Could you also refactor all of the parameter names. Various methods in `ResponsiveGrid` use 'column', 'columnName', 'columnHeader', or 'columnText'. I don't have a particular preference but we should pick one. I prefer `columnLabel` or `columnText` to indicate we are matching the display text on the column, not the database column name.

Reply:
columnLabel it is, my good man